utils/runtime.go

```diff
@@ -0,0 +1,45 @@
+package utils
+
+import (
+	"context"
+	sharedpb "github.com/tron-us/go-btfs-common/protos/shared"
+	"github.com/tron-us/go-common/db/postgres"
+	dbenv "github.com/tron-us/go-common/env/db"
+	"github.com/tron-us/go-common/log"
+	"go.uber.org/zap"
+)
+
+const (
+	DBConnectionHealthy = "DB Connection Healthy"
+	DBWriteConnectionError = "Cannot connect to the write database!"
+	DBReadConnectionError  = "Cannot connect to the read database!"
+)
+
+func CheckRuntime(ctx context.Context, runtime *sharedpb.RuntimeInfoRequest) (*sharedpb.RuntimeInfoReport, error) {
+
+	// db runtime
+	report := new(sharedpb.RuntimeInfoReport)
+
+	// Check postgres dbWrite
+	report.DbStatusExtra = []byte(DBConnectionHealthy)
+	report.Signature = runtime.Signature
+
+	PGDBWrite := postgres.CreateTGPGDB(dbenv.DBWriteURL)
```

Comment:
We did not have db connection every time, so check db connection should not be called every time on check runtime. Maybe you should put check db connection code in some other place, and ask the service call it separately.

Reply:
@JustinGaoF I agree. Since this is just a "utility" that will be used by multiple services, should we just leave the database connection part to the services themselves ? For example, in the main code of status-server, before it checks runtime (i.e., calls this utility), it should execute the database code. If the previous statement is true,  the only thing this utility will be doing is returning the 'report' object. Is that correct ? Ideas ? Concerns ?

Reply:
@uchetron the db check should still belong to this util, but you need either check the env if there's DBURL or set a flag to let the service toggle whether they have a database or not, same logic to redis check

Reply:
@wxue please check the latest. I have added a toggles with their respective environment variable so that the user can turn on or off both redis and postgres databases.